templates/trt-llm/packages/client.py

```diff
@@ -122,34 +56,99 @@ def load_server_and_model(self, env: dict):
             time.sleep(2)
             continue
 
-    @staticmethod
-    def stream_predict(user_data: UserData):
-        """Static method to yield predictions or errors based on input and a streaming user_data queue."""
+    def start_server(
+        self,
+        mpi: int = 1,
+        env: dict = {},
+    ):
+        """Triton Inference Server has different startup commands depending on
+        whether it is running in a TP=1 or TP>1 configuration. This function
+        starts the server with the appropriate command."""
+        def _build_server_start_command(
```

Comment:
nit: Seems like this function can be easily moved outside and doesn't need to be an inner function.